web_viewer/index.html

```diff
@@ -367,7 +367,9 @@
             const rrd_file = url_params.get('file') || 'colmap_fiat.rrd';
 
             // Normalize the extra slashes from the url
-            return (data_path + rrd_file).replace(/\/{2,}/g, '/');
+            const normalized = (data_path + rrd_file).replace(/\/{2,}/g, '/');
+            window.location.search = new URLSearchParams({ url: normalized }).toString();
```

Comment:
Is there any way to avoid URL-escaping here? This ends up looking like:
```
https://app.rerun.io/commit/4b3a728/index.html?url=%2Fdata%2Fcommit%2F4b3a728%2Fcolmap_fiat.rrd
```
which is a bit of an eye-sore.

Would be nice to end up with:
```
https://app.rerun.io/commit/4b3a728/index.html?url=/data/commit/4b3a728/colmap_fiat.rrd
https://app.rerun.io?url=/data/colmap_fiat.rrd
```

I think this might work?
```
url.search = decodeURIComponent(new URLSearchParams({ url: normalized }));
```

Reply:
If we don't want it encoded, then we might as well just use string interpolation

Reply:
![image](https://github.com/rerun-io/rerun/assets/1665677/d4aa1513-371e-40d0-a8f3-1b8af5418828)